.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: test
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - v1 # todo: delete this before releasing v1
+  pull_request:
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go-version: [1.16.x, 1.17.x]
+    name: go-test
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v2
+      # - name: go-test
```

Comment:
Uncomment?

Reply:
@mcdayoub is working on setting up the tests right now so he'll uncomment this with his PR. just putting it in place so it's super easy to get CI working once he's ready